### DISCUSSION TITLE
How to run telegram-chat-forward using Termux (Android) ?

### DISCUSSION THREAD

**Discussion by aahnik** [aahnik/tgcf#20] [Q&A] [answered]
Termux is an Android terminal emulator and Linux environment app.

**Comment by aahnik** [comment, marked as answer]
1. Install termux (Google Play Store or Fdroid)
2. Open termux and run `pkg install python git`
3. Now follow the steps from [here](https://github.com/aahnik/telegram-chat-forward/#installation) 
4. Run `pkg install micro` to install the text editor named **micro**
5. Now use `micro` to write configuration files.

**Reply by u-8d** [reply]
Sir i am having error here what can i do to solve i am using termux

https://user-images.githubusercontent.com/81148580/115363048-ba482000-a1df-11eb-981f-0dcb774f9271.jpg

screenshot

**Reply by aahnik** [reply]
wait for a few days. new version + documentation + video coming soon.

read #2 and #43

**Reply by u-8d** [reply]
Does few day means 1 august or in this or next because i need this very much 🙂

**Comment by u-8d** [comment]
Thank you sir

**Reply by aahnik** [reply]
bro, stop calling sir.

**Reply by u-8d** [reply]
Ok bro but when termux issue will be solved 😓

**Reply by aahnik** [reply]
there is actually no issue with termux. you most probably made some mistakes. The version you are using is an older one. Soon I will be releasing a new version along with documentation and videos. 

So please be patient.

in the mean time you can star and watch the project. also share with your friends,let this project reach 100 stars

**Reply by aahnik** [reply]
> Ok bro but when termux issue will be solved

Hi, @chiefgamer 
read this https://github.com/aahnik/tgcf/wiki/Run-on-Android-using-Termux

new version released

**Comment by u-8d** [comment]
Thanks, I'll check it out.
